core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -329,6 +335,28 @@ public LogicalPlan visitEval(Eval node, AnalysisContext context) {
     return new LogicalEval(child, expressionsBuilder.build());
   }
 
+  /**
+   * Build {@link LogicalHighlight}.
+   */
+  @Override
+  public LogicalPlan visitHighlight(Highlight node, AnalysisContext context) {
+    LogicalPlan child = node.getChild().get(0).accept(this, context);
```

Comment:
Just to confirm that `.get(0)` won't crash if there is no such element.

Reply:
I don't believe this case would be possible. As well the returning object at index `0` is standard for visit functions in the `Analyzer`.